armory/scenarios/multimodal_so2sat_scenario.py

```diff
@@ -0,0 +1,296 @@
+"""
+Multimodal image classification, currently designed for So2Sat dataset
+"""
+
+import logging
+from typing import Optional
+from copy import deepcopy
+
+from tqdm import tqdm
+import numpy as np
+
+from armory.utils.config_loading import (
+    load_dataset,
+    load_model,
+    load_attack,
+    load_adversarial_dataset,
+    load_defense_wrapper,
+    load_defense_internal,
+    load_label_targeter,
+)
+from armory.utils import metrics
+from armory.scenarios.base import Scenario
+
+logger = logging.getLogger(__name__)
+
+
+class So2SatClassification(Scenario):
+    def __init__(self, **kwargs):
+        if "attack_modality" not in kwargs.keys():
+            raise ValueError("`attack_modality` must be defined for So2Sat scenario")
+        if kwargs["attack_modality"] is None or kwargs["attack_modality"] not in (
```

Comment:
```suggestion
        if kwargs["attack_modality"] is None or kwargs["attack_modality"].lower() not in (
```

Reply:
Without `.lower()`, passing "SAR", "EO", or "Both", which is what the error message says is allowed, will actually raise an error